setup.py

```diff
@@ -24,31 +37,40 @@
     keywords='IRC parser data-analysis research development',
     packages=find_packages(exclude=['docs', 'tests']),
 
-    install_requires=[
-        'scipy',
-        'numpy',
+    dependency_links = ['https://github.com/rtfd/sphinx_rtd_theme/archive/0.2.4.tar.gz#egg=sphinx_rtd_theme-0.2.4'],
+
+	setup_requires = [
+        'graphviz'],
+
+    install_requires = [
         'networkx',
         'matplotlib',
+        'graphviz',
```

Comment:
we need to mention setup time dependencies and install time dependencies at both places.